Add the helpful hints for preinit/init/postinit.

WORMSS opened the pull request:
The list of what should happen where and when as mentioned by @Pahimar in his tutorial video.

Commit:
Add the helpful hints for preinit/init/postinit.

The list of what should happen where and when.

BrandonStaab commented:
I think at this time pahi only wants pull requests that will directly optimize or fix bug in the code, this has very little value as he will be explaining all of  this in his tutorials and for those who go a head of the series will have to reference EE3

WORMSS commented:
Yeah, well I added this as I had to keep replaying the same video video again and again to know where things should go. I added this to my own code in the end and thought others who are viewing his code online during his videos would benefit from it. As a visual recap of what he had said in the video.
